test/controllers/gkr_keyword_comparison_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class GkrKeywordComparisonControllerTest < ActionDispatch::IntegrationTest
+  test "should get show" do
+    get gkr_keyword_comparison_show_url
+    assert_response :success
+  end
```

Comment:
Instead of `get gkr_keyword_comparison_show_url` I would use `visit gkr_keyword_comparison_path` and verify that the form you created is being rendered properly. Additionally, you'll want to test that you can retrieve collections from CMR successfully. So maybe once your on the show page you can simulate someone filling out the form and then asserting that the proper collection is coming back from cmr

Reply:
Agree, so you'll want to create a erb that will be render the collection you pull from CMR, just for the purpose of testing, maybe you can add the abstract and keywords to it.   Then you can setup a test that verifies it is all working.